datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -118,56 +114,6 @@ where
             })
     }
 
-    fn logs_with_sigs(
-        &self,
-        logger: &Logger,
-        from: u64,
-        to: u64,
-        addresses: Vec<H160>,
-        event_signatures: Vec<H256>,
-    ) -> impl Future<Item = Vec<Log>, Error = Error> {
-        let eth_adapter = self.clone();
-        let logger = logger.to_owned();
-        let event_sig_count = event_signatures.len();
-
-        retry("eth_getLogs RPC call", &logger)
-            .no_limit()
-            .timeout_secs(60)
-            .run(move || {
-                // Create a log filter
-                let log_filter: Filter = FilterBuilder::default()
-                    .from_block(from.into())
-                    .to_block(to.into())
-                    .address(addresses.clone())
-                    .topics(Some(event_signatures.clone()), None, None, None)
-                    .build();
-
-                // Request logs from client
-                let logger = logger.clone();
-                eth_adapter
-                    .web3
-                    .eth()
-                    .logs(log_filter)
-                    .map(move |logs| {
-                        debug!(logger, "Received logs for blocks [{}, {}]", from, to);
-                        logs
-                    })
-                    .from_err::<EthereumContractCallError>()
-                    .from_err()
-            })
-            .map_err(move |e| {
-                e.into_inner().unwrap_or_else(move || {
```

Comment:
It looks like we're losing this error message. Is that deliberate?

Reply:
Yes, since this is retried indefinitely, this error can never be printed afaict.